Logger/BaseLoggerMixins.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+
+namespace Logger;
+
+public static class BaseLoggerMixins
+{
+    public static void Error(this BaseLogger baseLogger, string message, params object[] args)
+   { 
+        if (baseLogger == null) { throw new ArgumentNullException(nameof(baseLogger)); }
```

Comment:
This should be multiple lines
```suggestion
        if (baseLogger is null) 
        { 
           throw new ArgumentNullException(nameof(baseLogger)); 
        }
```